src/inventory/inventory.service.ts

```diff
@@ -81,32 +108,46 @@ export class InventoryService {
     return productItems;
   }
 
-  async findOne(_id: string) {
-    this.logger.debug(`{findOne} query: ${_id}`);
+  /**
+   * Finds a product item by its id.
+   * @param _id - The id of the product item to find.
```

Comment:
For me, the `id` field is an abstraction for graphql to be consistent with the other services.
Therefore, I use it in the resolver and Schema.
Internally (in the services and controllers) I use _id since it has nothing to do with graphql.
But maybe from an external POV `id` everywhere is easier to understand

Reply:
I mean I can foresee questions when someone else looks over this code.
But fine for now.

Reply:
I will point it out in an additional comment